### PR TITLE
add kirby-tidy-html plugin

### PR DESCRIPTION
**Pull request by dgsiegel**

### CODE REVIEW

**Comment by afbora** [comment]
Doesn't work the plugin for K3?

**Comment by dgsiegel** [comment]
> Doesn't work the plugin for K3?

It might, but I have never tested it :)

File: content/plugins/dgsiegel/kirby-tidy-html/plugin.txt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    'dgsiegel.kirby-tidy-html' => [
+        'enabled' => true,
+        'options' => [...]
+    ]
+];
+```
+
+----
+
+Versions: 4
```
Comment (afbora) [inline content/plugins/dgsiegel/kirby-tidy-html/plugin.txt]:
```suggestion
Versions: 3,4
```

**Review by afbora — APPROVED** [review]

**Comment by afbora** [comment]
In directory: https://getkirby.com/plugins/dgsiegel/kirby-tidy-html

**Pull request merged**